.travis.yml

```diff
@@ -1,6 +1,10 @@
 sudo: required
 dist: jammy
 
+branches:
+  only:
+    - dont-build
+
```

Comment:
Info - Can you explain this? What's the purpose of the "dont-build" branch?

Reply:
This was a low-risk way to disable the travis and appeveyor builds. 😸

Reply:
Before merging this PR, I will remove travis and appveyor files and disable them using settings on both platforms.

Reply:
Done!